cypress/integration-qe/e2e-createNewCluster.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('Flow', () => {
+  it('start from the /clusters page', () => {
+    // Set CYPRESS_BASE_URL environemnt variable
+    // Example: export CYPRESS_BASE_URL=http://localhost:3000
+    cy.visit('');
+  });
+
+  it('create a cluster named $CLUSTER_NAME', () => {
+    cy.get('#button-create-new-cluster').click();
+    cy.get('#form-input-name-field').should('be.visible');
+    cy.get('#form-input-name-field').clear();
+    cy.get('#form-input-name-field').type(Cypress.env('CLUSTER_NAME'));
```

Comment:
I think we should not use `Cypress.env()` directly in the test but copy values to a variable like it is done for `PULL_SECRET` in the `shared.js`.

In addition, we should provide reasonable defaults, if possible (see `cypress.json`).

Reply:
Done